src/Core/StaticConfigTrait.php

```diff
@@ -116,6 +107,72 @@ public static function config($key, $config = null)
     }
 
     /**
+     * Reads existing configuration.
+     *
+     * @param string $key The name of the configuration.
+     * @return array|null Array of configuration data.
+     */
+    public static function getConfig($key)
+    {
+        return isset(static::$_config[$key]) ? static::$_config[$key] : null;
+    }
+
+    /**
+     * This method can be used to define configuration adapters for an application
+     * or read existing configuration.
+     *
+     * To change an adapter's configuration at runtime, first drop the adapter and then
+     * reconfigure it.
+     *
+     * Adapters will not be constructed until the first operation is done.
+     *
+     * ### Usage
+     *
+     * Assuming that the class' name is `Cache` the following scenarios
+     * are supported:
+     *
+     * Reading config data back:
+     *
+     * ```
+     * Cache::config('default');
+     * ```
+     *
+     * Setting a cache engine up.
+     *
+     * ```
+     * Cache::config('default', $settings);
+     * ```
+     *
+     * Injecting a constructed adapter in:
```

Comment:
I'm not an English native speaker, but the ``in`` at the end seems superfluous, as you already have an ``in`` wthin ``Injecting``.